src/EditorFeatures/CSharpTest/Diagnostics/FixAllProvider/BatchFixerTests.cs

```diff
@@ -43,7 +43,7 @@ private static void AnalyzeNode(SyntaxNodeAnalysisContext context)
                     var symbol = context.SemanticModel.GetSymbolInfo(node).Symbol;
                     if (symbol != null && symbol.Kind == SymbolKind.Field)
                     {
-                        var diagnostic = CodeAnalysis.Diagnostic.Create(Descriptor, node.GetLocation());
+                        var diagnostic = Diagnostic.Create(Descriptor, node.GetLocation());
```

Comment:
These are reverts from the changes made in my previous PR to derive `AbstractCodeActionOrUserDiagnosticTest` from `TestBase`. That change has been reverted in this PR as well.

Reply:
sweet.